src/components/menuBar/js/menuItemDirective.js

```diff
@@ -20,10 +23,11 @@ function MenuItemDirective($mdUtil) {
         templateEl.html('');
         templateEl.append(angular.element('<md-icon md-svg-icon="check"></md-icon>'));
         templateEl.append(buttonEl);
-        templateEl[0].classList.add('md-indent');
+        templateEl.addClass('md-indent');
+        templateAttrs.$set('mdInMenuBar', null);
 
-        setDefault('role', (templateAttrs.type == 'checkbox') ? 'menuitemcheckbox' : 'menuitemradio', buttonEl);
-        angular.forEach(['ng-disabled'], moveAttrToButton);
+        setDefault('role', type == 'checkbox' ? 'menuitemcheckbox' : 'menuitemradio', buttonEl);
+        moveAttrToButton('ng-disabled');
```

Comment:
This function should be updated to use the `prefixer`.

To check if the attribute (with all prefixes) is present on the element.
```js
$mdUtil.prefixer().hasAttribute(element, 'ng-disabled');
```